plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/ProgramConverter.kt

```diff
@@ -38,6 +38,7 @@ class ProgramConverter(val session: FirSession, override val config: PluginConfi
     ProgramConversionContext {
     private val methods: MutableMap<MangledName, FunctionEmbedding> = SpecialKotlinFunctions.byName.toMutableMap()
     private val classes: MutableMap<MangledName, ClassTypeEmbedding> = mutableMapOf()
+    private val classDetails: MutableMap<ClassTypeEmbedding, ClassEmbeddingDetails> = mutableMapOf()
```

Comment:
Do we actually need `classDetails` when we have `details` property in every `ClassTypeEmbedding`?

Reply:
I guess there're issues with the initialization order, but I don't quite get them.

Reply:
The relevant situation here is when you get a class embedding but don't have associated details.  I guess in that case we could trust that the copy in `classes` does have an embedding, though... I'll think about it.

Reply:
I ended up removing this map, thanks for the suggestion :)